wpilibc/src/main/native/cpp/PIDController.cpp

```diff
@@ -652,7 +652,7 @@ void PIDController::InitSendable(SendableBuilder& builder) {
  * @return Error for continuous inputs.
  */
 double PIDController::GetContinuousError(double error) const {
-  if (m_continuous) {
+  if (m_continuous && m_inputRange != 0) {
```

Comment:
This should also check that `m_inputRange > 0` to match what is considered a valid range in Java.

Reply:
Since C++ doesn't enforce it on input like Java does, I wasn't sure this was the right place to enforce it.

Reply:
Actually, passing in a negative input range to `std::fmod()` seems to return the same value as a positive one. I guess the question is whether we should allow this or only perform the continuous mode calculation for positive ranges. Neither of those options appeal to me and we can't really inherit from ErrorBase due to PIDController already having a GetError() function.

Reply:
Created #884. I think I prefer ErrorBase, but that would require a breaking change so would have to be 2019 at the earliest.